src/__tests__/variables.test.ts

```diff
@@ -1577,6 +1588,39 @@ describe("Execute: Handles inputs", () => {
     });
   });
 
+  describe("Handles default values", () => {
```

Comment:
How is this test different from `"Execute: Uses argument default values.when no argument provided"`?

Reply:
Test uses input type that have field with default value. `"Execute: Uses argument default values.when no argument provided"` uses default value on basic type input instead. So the basic case where this issue comes is like this:

```
input TestInput {
  str: String
  type: String = "default"
}
type Query {
  test(input: TestInput!): String
}
```